rbc/heavydb/remoteheavydb.py

```diff
@@ -1266,8 +1268,35 @@ def _make_udf(self, caller, orig_sig, sig):
             name + sig.mangling(),
             atypes, rtype, annotations)
 
+    def _check_llvm_version(self):
+        # check LLVM version before compiling to LLVM
+        flag_name = 'RBC_DISABLE_LLVM_MISMATCH_WARN'
+        flag = int(os.environ.get(flag_name, False))
+
+        if not flag:
+            target = self.targets['cpu']
+            server_llvm_version = target.llvm_version
+            client_llvm_version = llvm.llvm_version_info
+
+            if (server_llvm_version[0], client_llvm_version[0]) == (11, 14):
+                c_llvm = '.'.join(map(str, client_llvm_version))
+                s_llvm = '.'.join(map(str, server_llvm_version))
+                msg = (
+                    f'The client LLVM version ({c_llvm}) is greater than the server '
+                    f'LLVM version ({s_llvm}). This is known to be unsupported. '
```

Comment:
With #551, I believe this statement has become out-of-date. Could you rephrase the message along the lines of `"If you experience any issues that could be due to the discrepancy of LLVM versions, please try to downgrade/update Numba that uses LLVM version ({s_llvm})."`. Also, this statement can be generalized to all LLVM version combinations, not just `(11, 14)`.

Reply:
I now see that the purpose is to warn only about the problematic LLVM version combinations. Since 11 and 14 are not problematic anymore, this function is essentially a no-op? What do you think?

In fact, the combination of 14 and 11 might be more problematic as a newer client-side LLVM version may use features (such as LLVM attributes or similar) that the server LLVM IR parser cannot handle.